CGPs/cgp-0107.md

```diff
@@ -0,0 +1,169 @@
+
+* Organized several workshops on smart contract and blockchain development for over 400+ web 3 enthusiasts and traditional developers. Some highlights below
+a. [Building your first Dapp](https://www.youtube.com/watch?v=L-Rz9MSYS9M)
+b. [How to build a voting application](https://youtu.be/THsM0hp9R7o?si=Ml2BbC2_kU7DkKMf)
+* Organized a [start-up pitch competition ](https://twitter.com/dappsoverapps/status/1692185380718538917?s=20) to encourage builders to integrate blockchain technology to their businesses and projects.
+* We organized the most participated web 3 art and cultural event that has onboarded 800+ creatives to web3. [Highlights from the recent edition ](https://www.youtube.com/watch?v=T6-46gcjIoo)
+
+**How do you know that people need your services?**
+
+* Visibility: < 1% of 85,000 developers in Nigeria are aware of Celo’s developer friendly infrastructure and its crypto mobile wallet solutions.
+* Branding Celo as the messiah of the good user experience that other web 3 applications lack for mass onboarding.
+* 9 out of 10 gen z’s in Africa are web 3 enthusiasts but lack the knowledge needed to kickstart their web 3 journey.
+* Traditional developers are looking for ways to transition into web3. Our hackathon and incubation program will be a turning point for developers in Africa.
+* Businesses across Africa need reliable payment solutions. Celo’s Valora wallet will be integrated into businesses in Africa.
+* Potential unicorn start-ups in Africa fail most of the time because of lack of support. We are dedicated to providing support to brilliant minds building amazing projects utilizing Celo’s blockchain.
+* Greener planet activists need solutions in addressing Climate change problem.
+
+**Grant received will be channeled into:**
+
+* Conferences and events: We will organize global conferences and monthly seminars in various cities and institutions in Africa. Kindly check out some of the videos and picture of some of the conferences we have hosted:
+* Workshop & Training for developers & non- developers on how to build with Celo. Kindly check out some of our technical workshops videos:
+* Organize startup pitch competition for project founders looking to build or integrate Celo’s infrastructure to their projects/start-ups.Kindly check out the one we hosted before
+* Organize hackathons and incubation programs focused on providing solutions to real life problems applicable to Africa and facing the world at large.
+* Guide project building on Celo to MVP, testnet and mainnet respectively.
+* Establish blockchain clubs in various institutions in Africa.
+* Create visibility via content creation and distribution.
+* Collaborate and consolidate efforts with various institutions and communities aligned with our goal.
+* Personal mentorship program with project founders, developers, tech enthusiasts and web3 newbies looking forward to building with Celo.
+
+|Particulars|KPIs & Deliverables|Budget-$63,400|Duration|
```

Comment:
Please add the Celo amount as the governace proposal payload is in CELO which fluctuates